conftest.py

```diff
@@ -152,19 +197,13 @@ def scratch_path(tmp_path, res_path, misc_info, is_win, unix2dos):
     # With the conversion of resources/objects_attrs.txt to Unix EOLs in order to
     # provide for a Unix-testable sdist, on Windows systems this resource needs
     # to be converted to DOS EOLs for consistency.
-    if is_win:
+    if is_win:  # pragma: no cover
```

Comment:
I'm confused as to why these lines started showing up in any of the coverage testing. My pipelines don't show either `setup.py` or `conftest.py`, in either the [GH Action][gha] for testing the main code or the [Azure Pipeline][ap] for checking execution of all the test code.

These pragmas shouldn't be necessary.

[ap]: https://dev.azure.com/brianskinn/sphobjinv/_build/results?buildId=564&view=logs&jobId=a97395d9-d56f-57fb-fb5c-9044ddefd9a7&j=a97395d9-d56f-57fb-fb5c-9044ddefd9a7&t=89437840-2e4d-5005-e67a-78711bfacfe0
[gha]: https://github.com/bskinn/sphobjinv/actions/runs/12458286225/job/34773701964#step:7:42